tests/validator/validator.cpp

```diff
@@ -1929,14 +1934,18 @@ TEST(Validator, unitEquivalenceExponentMultiplierPrefixExponent)
     libcellml::Variable::addEquivalence(v1, v2);
 
     validator->validateModel(model);
+
+    EXPECT_EQ(size_t(1), validator->issueCount());
     EXPECT_EQ(size_t(0), validator->errorCount());
+    EXPECT_EQ(size_t(0), validator->hintCount());
+    EXPECT_EQ(size_t(1), validator->warningCount());
+    EXPECT_EQ_ISSUES(expectedIssues, validator);
 }
 
 TEST(Validator, unitUserCreatedUnitsBananasAndApples)
 {
-    const std::vector<std::string> expectedErrors = {
-        "Variable 'v1' has units of 'bushell_of_apples' and an equivalent variable 'v2' with non-matching units of 'bunch_of_bananas'. The mismatch is: apple^10, banana^-5.",
-    };
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'v1' in component 'c1' has units of 'bushell_of_apples' and an equivalent variable 'v2' in component 'c2' with non-matching units of 'bunch_of_bananas'. The mismatch is: apple^10, banana^-5."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done.